core/org.eclipse.cdt.core/parser/org/eclipse/cdt/core/dom/ast/gnu/IGNUASTUnaryExpression.java

```diff
@@ -16,11 +16,22 @@
 import org.eclipse.cdt.core.dom.ast.IASTUnaryExpression;
 
 /**
- * @deprecated Replaced by {@link IASTUnaryExpression}.
  * @noreference This interface is not intended to be referenced by clients.
  */
-@Deprecated
```

Comment:
Removing the deprecation of this interface is unexpected. There is no concrete classes that implement this interface.